doc/man7/life_cycle-kdf.pod

```diff
@@ -81,8 +81,8 @@ This is the canonical list.
  EVP_KDF_CTX_reset                           newed        newed
  EVP_KDF_CTX_get_params                      newed       deriving
  EVP_KDF_CTX_set_params                      newed       deriving
- EVP_KDF_CTX_gettable_params                 newed       deriving
- EVP_KDF_CTX_settable_params                 newed       deriving
+ EVP_KDF_CTX_get_params                      newed       deriving
```

Comment:
IMO this change is incorrect - you have get/set_params above already. The original was IMO correct.